### PR TITLE
chore: v1.2.0 to main branch

### PR DESCRIPTION
**Pull request by veryCrunchy**
Allows us to remove `cli` branch and move on with working code.
(Code is based on `1.1.x` release, instead of broken `main` code)
New releases will have to be made manually again, as previous.

### CODE REVIEW

**Review by Bas950 — APPROVED** [review]

File: packages/cli/src/actions/create.ts [hunk 1]
```diff
@@ -228,7 +227,7 @@ console.log(prefix, chalk.green("Presence created! You can now start coding!"));
 
 async function serviceExists(service: string) {
 	try {
-		await access(`./websites/${getFolderLetter(service)}/${service}`);
+		await access(`./websites/${service.at(0)!.toUpperCase()}/${service}`);
```
Comment (Bas950) [inline packages/cli/src/actions/create.ts]: Why is it not using getFolderLetter?

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: packages/cli/src/actions/translate.ts [hunk 2]
```diff
@@ -152,7 +152,7 @@ async function translatePresences(presences: any, lang: string) {
 		if (translation === "stop") break;
 
 		const presencePath = resolve(
-			`./websites/${getFolderLetter(presence.service)}/${presence.service.replace("!", " ")}`
+			`./websites/${getFolderLetter(presence.service)}/${presence.service}`
```
Comment (Bas950) [inline packages/cli/src/actions/translate.ts]: pretty sure that replace is needed.

File: packages/cli/src/actions/create.ts [hunk 3]
```diff
@@ -181,7 +181,7 @@ const res = await inquirer.prompt<{
 ]);
 
 const presencePath = resolve(
-	`./websites/${getFolderLetter(res.service)}/${res.service.replace("!", " ")}`
+	`./websites/${getFolderLetter(res.service)}/${res.service}`
```
Comment (Bas950) [inline packages/cli/src/actions/create.ts]: ditto

File: packages/cli/src/actions/modify.ts [hunk 4]
```diff
@@ -1,45 +1,258 @@
...
+  ) {
+    console.log(prefix, chalk.redBright("Could not find presence:", service));
+    process.exit(0);
+  }
+}
 
 const require = createRequire(import.meta.url);
 const presencePath = resolve(
-	`./websites/${getFolderLetter(service)}/${service.replace("!", " ")}`
+  `./websites/${getFolderLetter(service)}/${service}`
```
Comment (Bas950) [inline packages/cli/src/actions/modify.ts]: ditto

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: packages/cli/src/actions/create.ts (hunk 1 above)
Comment (veryCrunchy) [reply packages/cli/src/actions/create.ts]: Code is based of the previous release. So it was never used in there.
Will update and test tomorrow.

**Comment by veryCrunchy** [comment]
~~@Bas950 should we not replace with "" instead of " "?~~
We definitely should
image

**Comment by Bas950** [comment]
> ~~@Bas950 should we not replace with "" instead of " "?~~
> We definitely should
> image
> 

I think it got trimmed afterwards

**Comment by veryCrunchy** [comment]
> I think it got trimmed afterwards

Ah, right. trim

**Review by Bas950 — CHANGES_REQUESTED** [review]
you need to replace ! with " " (with a space) otherwise `osu!ppy` doesnt turn into `osu ppy` keeping the trim for when its at the start or end is correct.

**Comment by veryCrunchy** [comment]
mbmb

**Review by Bas950 — APPROVED** [review]

**Pull request merged**